rb/spec/integration/selenium/webdriver/firefox/profile_spec.rb

```diff
@@ -22,137 +22,161 @@
 module Selenium
   module WebDriver
     module Firefox
-      compliant_on driver: :ff_legacy do
```

Comment:
I think we still need to specify that this is only compliant on `:firefox` & `:ff_legacy`. We can use `browser:` instead of `driver:` if we can use it local and remote.